draft-beky-httpbis-metadata.md

```diff
@@ -73,7 +85,7 @@ list of key-value pairs. Each key and value is a sequence of bytes with no
 restriction on the allowed values. The encoded block is packaged as the payload
 of one or more frames.
 
-An endpoint _MAY_ transmit multiple metadata blocks on the same stream.
+An endpoint MAY transmit multiple metadata blocks on the same stream.
```

Comment:
What is the reasoning for the removal of emphasis in the various places here and below? Just editorial taste, or some other criteria?

Reply:
The tools automatically add emphasis, so we should follow the usual practice to make sure we get the automatic tooling